dali/test/python/test_pipeline.py

```diff
@@ -30,6 +30,8 @@
 from test_utils import check_batch
 from test_utils import compare_pipelines
 
+from nose.tools import assert_raises
```

Comment:
What is that for?

Reply:
Here it is a leftover from previous version. Done.